src/dioptra/restapi/v1/workflows/schema.py

```diff
@@ -41,3 +43,81 @@ class JobFilesDownloadQueryParametersSchema(Schema):
         by_value=True,
         default=FileTypes.TAR_GZ.value,
     )
+
+
+class ResourceImportSourceTypes(Enum):
+    GIT = "git"
+    UPLOAD = "upload"
+
+
+class ResourceImportResolveNameConflictsStrategy(Enum):
+    FAIL = "fail"
+    OVERWRITE = "overwrite"
+
+
+class ResourceImportSchema(Schema):
+    """The request schema for importing resources"""
+
+    groupId = fields.Integer(
+        attribute="group_id",
+        # data_key="group",
```

Comment:
I think this is a bug in marshmallow swagger docs. When the schema is used as form data, it ignores `data_key` in the swagger UI.